src/components/Heading/Heading.tsx

```diff
@@ -17,6 +18,8 @@ export type Props = {
   tag?: HeadingTagTypes
   /** コンポーネントに適用するクラス名 */
   className?: string
+  /** 非表示のHeadingにするフラグ */
```

Comment:
WANT: 視覚的に非表示であることを伝えたい

```suggestion
  /** 視覚的に非表示にするフラグ */
```